RobotArmAPP/Classes/HTTPRequests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace RobotArmAPP.Classes
+{
+    class HTTPRequests
+    {
+        public async Task<string> GetRequest(string eixo1Valor, string eixo2Valor, string eixo3Valor, string eixo4Valor, string garraValor)
```

Comment:
Uma questão de nomeação. Para métodos de fora da classe, pouco importa qual método http você está usando. Assim, sugiro um nome que descreva efetivamente o que você está fazendo
```suggestion
        public async Task<string> SendMovementToRobot(Movement movement)
```

Reply:
Feito!